data/talkactions/scripts/start_cast.lua

```diff
@@ -0,0 +1,9 @@
+function onSay(cid, words, param)
+	local player = Player(cid)
```

Comment:
I'd consider adding a little more informative error handling in the ProtocolGame::startLiveCast method in the future. Which would unnecessarily complicate the talkaction (these commands are supposed to be a reference implementation, it's quite easy to implement it the way you're suggesting right now)